cloudify_agent/installer/__init__.py

```diff
@@ -341,16 +342,21 @@ def create_custom_env_file_on_target(self, environment):
             return None
 
     def download(self, url, destination=None):
+        local_cert_file = self.cloudify_agent.get('local_rest_cert_file')
+        verify_cert = self.cloudify_agent.get('verify_rest_certificate')
+        if verify_cert is None:
```

Comment:
but according to 
https://github.com/cloudify-cosmo/cloudify-agent/blob/windows-agent-cert/cloudify_agent/installer/config/configuration.py#L207l
this is always set. configuration runs before download, daemons runs after